apps/api_web/lib/api_web/controllers/route_pattern_controller.ex

```diff
@@ -10,7 +10,7 @@ defmodule ApiWeb.RoutePatternController do
 
   plug(:ensure_path_matches_version)
 
-  @filters ~w(id route direction_id stop)
+  @filters ~w(id is_canonical route direction_id stop)
```

Comment:
question: why did we go with `is_canonical` vs only `canonical`? The other boolean we have is `timepoint` on `/schedules` and it doesn't have the `is_` prefix.

Reply:
No objections here to dropping the `is_` prefix. 🙂